app/components/school-vocabularies-expanded.js

```diff
@@ -31,9 +31,9 @@ export default class SchoolVocabulariesExpandedComponent extends Component {
   }
 
   @restartableTask
-  *load(element, [school]) {
-    yield this.loadSchool(school.id);
-    const vocabularies = (yield school.vocabularies).toArray();
+  *load() {
```

Comment:
I don't see where this is triggered, am I missing it in the template?

Reply:
it's wired up in the corresponding template:
https://github.com/ilios/frontend/blob/master/app/components/school-vocabularies-expanded.hbs#L3

Reply:
Ah, Github was hiding the top of that file. Sorry for the noise!